reference/configuration/framework.rst

```diff
@@ -647,6 +649,24 @@ service
 
 This setting contains the service id of a custom matcher.
 
+request
+~~~~~~~
+
+formats
```

Comment:
Should we include here the configuration example of the original `request/mime_type.rst` article?

Reply:
good idea - done!